src/utils/Processing.ts

```diff
@@ -201,9 +208,19 @@ function findNonDuplicateSecondaryFiles(files: string[], docs: WildFHIR[]): stri
 
 function readJSONorXML(file: string): any {
```

Comment:
This is pretty small, but can the return type of this function be `FileImport` now?

Reply:
It certainly can be! Addressed in commit `969399b`.